src/devices/machine/smioc.h

```diff
@@ -0,0 +1,63 @@
+// license:GPL-2.0+
+// copyright-holders:Brandon Munger
+/**********************************************************************
+
+    ROLM 9751 9005 System Monitor Input/Ouput Card emulation
+
+**********************************************************************/
+
+#ifndef MAME_MACHINE_SMIOC_H
+#define MAME_MACHINE_SMIOC_H
+
+#pragma once
+
+#include "cpu/i86/i186.h"
+
+//**************************************************************************
+//  MACROS / CONSTANTS
+//**************************************************************************
+
+#define SMIOC_TAG           "smioc"
```

Comment:
Expecting a device to always have the same tag is dangerous.  Please don't encourage it by placing macros like this in device headers.